recipes/vaex-core/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "vaex-core" %}
+{% set version = "0.1.3" %}
+{% set sha256 = "07633b513081f27a2eb266a419914961c24f517885b8c1a3e572a3ddac7a196f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+     - vaex = vaex.__main__:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy 1.11.*
+    - toolchain
+  run:
+    - python
+    - numpy >=1.11
+    - scipy
+    - astropy >=1
+    - aplus
+    - futures >=2.2.0  # [py27 or py35]
```

Comment:
Interesting that this is needed on Python 3.5, but not Python 3.6. 🤔

Reply:
Hah, yeah, it was actually needed for a different recipe (most likely some bug when 3.6 was just new orso). But I'll try without.